requirements.txt

```diff
@@ -0,0 +1,19 @@
+docutils==0.15.2
```

Comment:
I think this is only needed for building the docs

Reply:
Removed.